clang/test/CIR/CodeGen/OpenCL/spirv-target.cl

```diff
@@ -0,0 +1,29 @@
+// See also: clang/test/CodeGenOpenCL/spirv_target.cl
+// RUN: %clang_cc1 -cl-std=CL3.0 -fclangir -emit-cir -triple spirv64-unknown-unknown %s -o %t_64.cir
```

Comment:
I also tried SPIRV32 but got wrong alignment of struct fields. Maybe related to MLIR data layout. Investigating it, will file an issue if it's general.

Reply:
If you got wrong alignment you'd probably need to look at it anyways at some point? Creating an issue and assigning to yourself sounds like a good idea.